content/lang/checklists/checklistloader.en.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/*
+------------------
+Language: English
+------------------
+*/
+
+$LANG['SELECT_FILE'] = 'Please select a file to upload';
+$LANG['DOCUMENT'] = 'Document ';
```

Comment:
I find it's best to leave the trailing spaces in the primary file rather than including them in the language file. They are more likely to be accidentally omitted upon translation when they're in the language file.